Dockerfile

```diff
@@ -2,11 +2,18 @@ FROM python:3.9-buster
 ARG PYTHON_REQUIREMENTS=requirements.txt
 ENV PYTHONUNBUFFERED=1
 ENV DOCKERIZE_VERSION v0.6.1
+ENV NODE_MAJOR 18
 ARG BUILD_PG_MAJOR=15
 ENV PG_MAJOR=$BUILD_PG_MAJOR
+
+# Install dependencies
+RUN apt-get update
```

Comment:
It would be good to remove the ` /var/lib/apt/lists/*; \` using `rm -rf /var/lib/apt/lists/*; \` to avoid bloating the docker image.

Reply:
Makes sense. I think we could do that later when focusing on reducing the docker image size.